cpp/src/Ice/Incoming.cpp

```diff
@@ -31,50 +29,42 @@ extern bool printStackTraces;
 
 }
 
-Ice::MarshaledResult::MarshaledResult(const Ice::Current& current) :
```

Comment:
Indentation looks wrong.

Reply:
This is all going to be fixed soon once we apply (and keep applying) clang format.